src/Package/Fetch.zig

```diff
@@ -512,6 +532,8 @@ fn runResource(
         ) });
         return error.FetchFailed;
     };
+    // Remove temporary directory root.
+    cache_root.handle.deleteTree(tmp_dir_sub_path) catch {};
```

Comment:
1. Syscall should be avoided when it is known to not exist already.
2. When it is needed, it is known to be an empty directory, so it is better to use [deleteDir](https://ziglang.org/documentation/master/std/#std.fs.Dir.deleteDir).